lib/location/scan_wifi.c

```diff
@@ -112,6 +112,9 @@ void scan_wifi_execute(int32_t timeout, struct k_sem *wifi_scan_ready)
 #if defined(CONFIG_LOCATION_METHOD_WIFI_NET_IF_UPDOWN)
 	ret = scan_wifi_startup_interface(wifi_iface);
 	if (ret) {
+		LOG_ERR("Failed to start Wi-Fi interface: %d", ret);
```

Comment:
This is a duplicate log to what `scan_wifi_startup_interface` logs, right?

Reply:
I do agree in general. But when calling a static function, one log from that call chain should be enough.